### PR TITLE
vizer: a paraview/trame based web application

### PR DESCRIPTION
**Pull request by utkarshayachit**
Adds a ParaView/trame based web application as a demo.

### CODE REVIEW

**Comment by utkarshayachit** [comment]
@xpillions, this is ready for review. thanks.

File: playbooks/roles/ood-applications/files/bc_vizer/form.yml.erb [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        id: "container_image"
+        label: "Container Image"
+        widget: "text_field"
+        value: "utkarshayachit/vizer:${gl}"
+        required: true
+        help: |
+            Container image to use for the application server. Use `${gl}` as a placeholder
+            for the OpenGL implementation. It will be replced with `egl` or `osmesa` based on
+            the selected node type.
+    dataset_dir:
```
Comment (xpillons) [inline playbooks/roles/ood-applications/files/bc_vizer/form.yml.erb:64]: There is a way to allow a folder to be selected by the file explorer. Please have a look to the working_dir of the bc_codeserver app. You need to also add the form.js file

**Review by xpillons — CHANGES_REQUESTED** [review]
Would it be possible to also automatically download an example during the installation ?

**Comment by utkarshayachit** [comment]
> Would it be possible to also automatically download an example during the installation ?

done.

**Comment by utkarshayachit** [comment]
@xpillions, I've made the updates. thanks.

**Review by xpillons — APPROVED** [review]

**Comment by utkarshayachit** [comment]
thanks for the review!

**Pull request merged**